presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -722,6 +723,90 @@ public void testSchemaMismatch()
                 .isFailingForPageSource(new ParquetPageSourceFactory(TYPE_MANAGER, HDFS_ENVIRONMENT, STATS, new HadoopFileOpener()), expectedErrorCode, expectedMessageRowLongNest);
     }
 
+    @Test
+    public void testSchemaMismatchOnNestedStruct()
+            throws Exception
+    {
+        //testing fields order does NOT matter
```

Comment:
test out of order fields in nested Row type

Reply:
done